View/Screens/ArtistScreen.xaml.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Mail;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media.Imaging;
+using Controller;
+using Controller.DbControllers;
+using Microsoft.Win32;
+using Model.Database.Contexts;
+using Model.DbModels;
+using Model.MailTemplates;
+using View.DataContexts;
+
+namespace View.Screens
+{
+    partial class ArtistScreen : ResourceDictionary
+    {
+        public ArtistScreen()
+        {
+            InitializeComponent();
+        }
+
+        private void UploadSongButton_Click(object sender, RoutedEventArgs e)
+        {
+            var fileDialog = new OpenFileDialog {DefaultExt = ".mp3", Filter = "MP3 Files (.mp3)|*.mp3"};
```

Comment:
Is er een reden waarom we alleen .mp3 accepteren?

Reply:
volgens mij niet, we kunnen ook andere formaten doen